### PR TITLE
Events Page Hero Section + bug fixes

### PR DESCRIPTION
**Pull request by Navojith**

### CODE REVIEW

File: src/components/client/eventsPage/HeroSection/index.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <div className="row">
+          <h1 id="events-hero-title">{title}</h1>
+        </div>
+        <div className="row">
+          <div className="col-12 col-sm-12 col-md-6 col-lg-6">
+            <p className="desc">{description}</p>
+          </div>
+          <div className="col-12 col-sm-12 col-md-6 col-lg-6 d-flex justify-content-end align-items-center mt-4 mt-lg-0 mt-md-4 mt-sm-4">
+            <div id="events-hero-img-container">
+              <img id="events-hero-img" src={imgSrc} />
```
Comment (senuravihanjayadeva) [inline src/components/client/eventsPage/HeroSection/index.jsx]: add alt text

File: src/routes/index.jsx [hunk 2]
```diff
@@ -10,6 +11,7 @@ function RouteComponent() {
       <Routes>
         <Route path="/home" element={<LandingPage />} />
         <Route path="/" element={<LandingPage />} />
```
Comment (senuravihanjayadeva) [inline src/routes/index.jsx]: put this at the start.
Exact path matches should be placed before less specific path matches.Wildcard matches should be placed last, after all other path matches.

File: src/components/client/eventsPage/HeroSection/index.jsx (hunk 1 above)
Comment (senuravihanjayadeva) [reply src/components/client/eventsPage/HeroSection/index.jsx]: There are several reasons why adding alt text to the "img" tag is important:

Accessibility: Adding alt text to an image is essential for making your content more accessible to people with visual impairments or those who rely on screen readers. When an image has an alt text, the screen reader will read out the description to the user, providing them with a better understanding of what the image contains.

SEO: Search engines use alt text to understand what the image is about, which can improve the visibility of your website in search engine results pages (SERPs). Without alt text, search engines may have a hard time understanding what your content is about and may not index it properly.

Placeholder: If an image fails to load, the alt text will be displayed instead, providing context to the user.

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]

File: src/components/client/eventsPage/HeroSection/index.jsx (hunk 1 above)
Comment (Navojith) [reply src/components/client/eventsPage/HeroSection/index.jsx]: oh I didnt know that. thank you. I will add that too

**Comment by uihalagedara** [comment]
LGTM

**Review by uihalagedara — APPROVED** [review]

**Pull request merged**